docs/actions.md

```
@@ -169,12 +169,15 @@ Executes the following steps:
     skip-no-healthcheck: false
 ```
 
-**Note**: The usage of `skip-no-healthcheck: true` is only suggested when an image inside your stack does not provide a healthcheck and also the [definition of a custom healthcheck](https://github.com/peter-evans/docker-compose-healthcheck) is not possible.
-This could be e.g. the case when a barebone Unix image (like `alpine`) is used and tools like `wget` or `curl` are missing.
+**Note**: The usage of `skip-no-healthcheck: true` is only suggested when an image inside your stack does not provide a
+health check and also the [definition of a custom healthcheck](https://github.com/peter-evans/docker-compose-healthcheck)
+is not possible. This could be e.g. the case when a barebone Unix image (like `alpine`) is used and tools like `wget` 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)